### DISCUSSION TITLE
SSO class 'jwt.exceptions.ImmatureSignatureError'

### DISCUSSION THREAD

**Discussion by crato-ctl** [netbox-community/netbox#10846] [General]
I m using SSO
![Sin título](https://user-images.githubusercontent.com/74680992/200017679-c2cd0d98-2d33-4ac1-acc4-3c8ace90d396.png)

**Comment by jcollie** [comment]
Just a guess, but make sure that the clock on your server is correct.

**Reply by crato-ctl** [reply]
Thanks for your quick response, now i have the following
![image](https://user-images.githubusercontent.com/74680992/200032316-fdced60b-4328-4c44-af70-183a8f9060d6.png)

**Reply by jcollie** [reply]
You're beyond my skills with SSO. I'd suggest posting your SSO config here (redact secrets of course) for more help.

**Reply by 0xbbeer** [reply]
> Just a guess, but make sure that the clock on your server is correct.

Thank you! I have same problem with keycloak. For some reason, ntp was not configured. The lag was not noticeable, and amounted to hundredths of a second

**Comment by crato-ctl** [comment]
thanks again jcollie, 
this is the  configuration file cat /opt/netbox-3.3.5/netbox/netbox/configuration.py
![image](https://user-images.githubusercontent.com/74680992/200042066-d9a4dd75-9a42-4cf7-91f1-bfdb95e43875.png)
![image](https://user-images.githubusercontent.com/74680992/200042493-765b6d40-5f44-4b6d-8db2-20808ec0a42d.png)
![image](https://user-images.githubusercontent.com/74680992/200042683-dd790883-dff4-4186-ad94-d22954419e5d.png)

i dont know if you need any thing else. thanks!!
